QModInstaller/QModPatcher.cs

```diff
@@ -316,19 +320,29 @@ public static string Blank(int maxLength)
             return output;
         }
     }
-
-    /*class Patches
+    class Patches
     {
         [HarmonyPatch(typeof(UIScreenBugReport))]
-        [HarmonyPatch("PostIt")]
+        [HarmonyPatch("Post")]
         class UIScreenBugReport_PostIt
         {
-            [HarmonyTranspiler]
-            [HarmonyPriority(int.MinValue)]
-            static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstruction> instr)
+            static void Prefix(UIScreenBugReport __instance)
             {
-                return null;
+                UnityEngine.Debug.Log("Bug report UI screen sending!");
+                var m_Body = Traverse.Create(__instance).Field("m_Body");
+                var Text = m_Body.Property("text");
+                var text = m_Body.Field("m_Text");
+                string value = text.GetValue<string>();
+                if (!value.ToLower().Contains("this game is modded"))
+                {
+                    value += "This game is modded\n\n";
+                    UnityEngine.Debug.Log("Couldn't find modded flag! Adding");
+                    Text.SetValue(value);
+                    text.SetValue(value);
+                    Text.SetValue(value);
+                    text.SetValue(value);
```

Comment:
Why is each line called two times?

Reply:
That's because I can't trust it running once.